cg/apps/scout/scout_export.py

```diff
@@ -47,6 +56,13 @@ class Gene(BaseModel):
     polyphen_prediction: Optional[str]
 
 
+class DiagnosisPhenotypes(BaseModel):
+    disease_nr: int
+    disease_id: str
+    description: str
+    individuals: Optional[List[Dict[str, str]]]
```

Comment:
Exactly, this is optional. The whole thing in the scoiut database looks like this:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/28093618/164384773-14f85f65-d5a9-4d8f-997b-8248955441f3.png">